### ISSUE TITLE
Error in raxml_sliding_windows.py

### ISSUE THREAD

**Issue by rancilhac** [simonhmartin/genomics_general#69]
Hello,

I am trying to run the script raxml_sliding_windows.py in python 3.8, but I always get the following error:

`$ python raxml_sliding_windows.py -h
  File "raxml_sliding_windows.py", line 329
    print "\nDone."
          ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print("\nDone.")?
`
Is this a known bug?

Best regards,
Loïs Rancilhac

**Comment by simonhmartin** [comment]
Yes this script has still not been converted to Python 3 syntax. I will try to get to that soon. In the meantime, if you can install python 2, it should run.

**Comment by rancilhac** [comment]
Thanks a lot for a very quick reply. Using python 2 solved the problem indeed.
